tap_shopify/streams/inventory_levels.py

```diff
@@ -10,9 +10,12 @@ class InventoryLevels(Stream):
     replication_key = 'updated_at'
     key_properties = ['location_id', 'inventory_item_id']
     replication_object = shopify.InventoryLevel
+    # Added decorator over functions of shopify SDK
+    replication_object.find = shopify_error_handling(replication_object.find)
 
-    @shopify_error_handling
```

Comment:
Why do we remove this, isn't it required for other error handling?

Reply:
As we have added the decorator over the `find` method. Hence removed this decorator.